docs/reference/cli/push.mdx

```diff
@@ -18,6 +18,9 @@ Push the truss as a published deployment. If no production deployment exists, pr
 <ParamField body="--promote" type="BOOL">
 Push the truss as a published deployment. Even if a production deployment exists, promote the truss to production after deploy completes.
 </ParamField>
+<ParamField body="--preserve-previous-production-settings" type="BOOL">
```

Comment:
let's make sure this & all of the variable naming matches the new name `--preserve-previous-production-deployment`

Reply:
oh yeah, my bad, I didn't realize we changed `settings` for `deployment`. I'll make sure that the correct naming is used everywhere